src/main/webapp/app/exam/examExercisesNavigationService.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+
+@Injectable({ providedIn: 'root' })
+export class ExamExercisesNavigationService {
+    private exerciseIdToNavigateToSource = new BehaviorSubject(-1);
+    currentExerciseIdToNavigateTo = this.exerciseIdToNavigateToSource.asObservable();
+
+    constructor() {}
+
+    navigateToExamExercise(exerciseId: number) {
+        this.exerciseIdToNavigateToSource.next(exerciseId);
+    }
+}
```

Comment:
Yes, my mistake.
I now changed it to singular. I had in mind that you use this service to navigate between different (multiple) exercises and not navigate inside one exercise, that's why I chose the plural form. What do you think about this? @lschlesinger 

Could you recommend a subdirectory where I should/could put this service? Because the exam and notification files are quite different, but two components (one from each side/domain) should use this service.
I thought that this service is only used during the exam (mode) but the notification-side will not use it all the time, that's why I put it inside the exam folder.

Reply:
I think singular is fine, my association is still navigating from one exercise to another. I don't know the code base at this area very well... but currently the file looks lost. Maybe put it in the manage subfolder (as it is a management task to notify participants on changes during an exam) and call it `exam-exercise-update.service.ts` to make it as consistent as possible with the other names (live is maybe not need, if so, than I would call it `exam-exercise-live-update.service.ts`)

Reply:
Good idea. Done.